tests/test_utils/test_version.py

```diff
@@ -0,0 +1,3 @@
+import linkml_runtime
+
+print(linkml_runtime.__version__)
```

Comment:
can you make it an actual test (and avoid prints)?

Reply:
Updated the unit test.